packages/sync-actions/test/customer-group-sync.spec.js

```diff
@@ -1,38 +1,136 @@
 import customerGroupSyncFn, { actionGroups } from '../src/customer-group'
 import { baseActionsList } from '../src/customer-group-actions'
 
-describe('Exports', () => {
+describe('Discount Groups Exports', () => {
   it('action group list', () => {
-    expect(actionGroups).toEqual(['base'])
+    expect(actionGroups).toEqual(['base', 'custom'])
   })
 
-  it('correctly defined based actions list', () => {
-    expect(baseActionsList).toEqual([
-      { action: 'changeName', key: 'name' },
-      { action: 'setKey', key: 'key' },
-    ])
+  describe('action list', () => {
```

Comment:
You should also assert for the `setCustomType` and `setCustomField` actions

Reply:
Nope. They're not part of the actionsBaseList.

Reply:
Ah yes. Missed that 💯